app.js

```diff
@@ -1,28 +1,44 @@
+/* eslint-disable no-console */
 const express = require("express");
 const { graphqlExpress, graphiqlExpress } = require("graphql-server-express");
+const { addErrorLoggingToSchema } = require("graphql-tools");
 const repositories = require("./repositories");
 const colors = require("colors");
 const cors = require("cors");
 const bodyParser = require("body-parser");
 const schema = require("./schema");
-const morgan = require("morgan");
-
+const pino = require("express-pino-logger")();
 const { PORT } = require("./config/settings");
 
 const app = express();
-app.use(morgan("tiny"));
+
+app.use(pino);
+
+const formatError = e => {
```

Comment:
This is badly named. Seems better to extract the `logger` object below, and add a `log` method to it?

Reply:
Only reason for naming it `formatError` was because it's passed into `graphqlExpress` options. Would something like this be better?

```
const log = error =>{
     console.error(error);
     return e;
} 
const logger = {
 log
}
```

then

`..., formatError: logger}`